### PR TITLE
Update Nubiles.yml

### PR DESCRIPTION
**Pull request by aussiehuddo**
Added multiple sites, edited gallery section too.

### CODE REVIEW

File: scrapers/Nubiles.yml [hunk 1]
```diff
@@ -17,7 +17,14 @@ sceneByURL:
       - detentiongirls.com/video/watch/
       - driverxxx.com/video/watch/
       - familyswap.xxx/video/watch/
+      - imnotyourmommy.com/video/watch/
+      - momsboytoy.com/video/watch/
```
Comment (Belleyy) [inline scrapers/Nubiles.yml:21]:
```suggestion
      - momlover.com/video/watch/
      - momsboytoy.com/video/watch/
```

File: scrapers/Nubiles.yml [hunk 2]
```diff
@@ -57,11 +64,19 @@ galleryByURL:
     url:
       - badteenspunished.com/photo/gallery/
       - bountyhunterporn.com/photo/gallery/
+      - brattymilf.com/photo/gallery/
       - daddyslilangel.com/photo/gallery/
       - detentiongirls.com/photo/gallery/
       - driverxxx.com/photo/gallery/
       - familyswap.xxx/photo/gallery/
+      - imnotyourmommy.com/photo/gallery/
+      - momsboytoy.com/photo/gallery/
```
Comment (Belleyy) [inline scrapers/Nubiles.yml:73]:
```suggestion
      - momlover.com/photo/gallery/
      - momsboytoy.com/photo/gallery/
```

**Comment by Belleyy** [comment]
Added momlover site as well. You need to update [SCRAPERS-LIST.md](https://github.com/stashapp/CommunityScrapers/blob/master/SCRAPERS-LIST.md) as well.

**Comment by bnkai** [comment]
> Added momlover site. You need to update [SCRAPERS-LIST.md](https://github.com/stashapp/CommunityScrapers/blob/master/SCRAPERS-LIST.md) as well.

I think a separate PR was made <https://github.com/stashapp/CommunityScrapers/pull/1003> with the list changes for  #1004 #1005. Not oprimal but should be ok

**Comment by aussiehuddo** [comment]
> > Added momlover site. You need to update [SCRAPERS-LIST.md](https://github.com/stashapp/CommunityScrapers/blob/master/SCRAPERS-LIST.md) as well.
> 
> I think a separate PR was made #1003 with the list changes for #1004 #1005. Not oprimal but should be ok

this is correct. Sorry I am new to github and use the web editor.

File: scrapers/Nubiles.yml (hunk 1 above)
Comment (aussiehuddo) [reply scrapers/Nubiles.yml:23]: already added

File: scrapers/Nubiles.yml [hunk 3]
```diff
@@ -17,7 +17,16 @@ sceneByURL:
       - detentiongirls.com/video/watch/
       - driverxxx.com/video/watch/
       - familyswap.xxx/video/watch/
+      - imnotyourmommy.com/video/watch/
+      - momlover.com/video/watch/
+      - momlover.com/video/watch/
```
Comment (aussiehuddo) [inline scrapers/Nubiles.yml:22]:
```suggestion
      - momlover.com/video/watch/
```

File: scrapers/Nubiles.yml [hunk 4]
```diff
@@ -57,11 +65,21 @@ galleryByURL:
...
       - badteenspunished.com/photo/gallery/
       - bountyhunterporn.com/photo/gallery/
+      - brattymilf.com/photo/gallery/
       - daddyslilangel.com/photo/gallery/
       - detentiongirls.com/photo/gallery/
       - driverxxx.com/photo/gallery/
       - familyswap.xxx/photo/gallery/
+      - imnotyourmommy.com/photo/gallery/
+      - momlover.com/photo/gallery/
+      - momlover.com/photo/gallery/
```
Comment (aussiehuddo) [inline scrapers/Nubiles.yml:75]:
```suggestion
```

**Comment by aussiehuddo** [comment]
#1003 and #1005 should now match

**Pull request merged**